JWT/JWT.cs

```diff
@@ -173,6 +173,25 @@ private static void Verify(string decodedCrypto, string decodedSignature, string
                     throw new SignatureVerificationException("Token has expired.");
                 }
             }
+            // verify nbf claim https://tools.ietf.org/html/draft-ietf-oauth-json-web-token-32#section-4.1.5
+            if (payloadData.ContainsKey("nbf") && payloadData["nbf"] != null)
```

Comment:
Yeah, consistency makes sense. I submitted a refactoring PR but the author never cared to review it.
P.S. you still can have proper exception by splitting the if condition into 2:
```csharp
object nbfObj;
if (payloadData.TryGetValue("nbf", out nbfObj))
{
    if (!nbfObj is int)
        throw new SignatureVerificationException("Claim 'nbf' must be an integer.");
    int nbf = (int)nbfObj;
}
```